src/app/tests/suites/TestTemperatureControl.yaml

```diff
@@ -0,0 +1,71 @@
+# Copyright (c) 2021 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Temperature Control Cluster Tests
+
+config:
+    nodeId: 0x12344321
+    cluster: "Temperature Control"
+    endpoint: 1
+
+tests:
+    - label: "Wait for the commissioned device to be retrieved"
+      cluster: "DelayCommands"
+      command: "WaitForCommissionee"
+      arguments:
+          values:
+              - name: "nodeId"
+                value: nodeId
+
+    - label: "Read supported temperature levels"
+      command: "readAttribute"
+      attribute: "SupportedTemperatureLevels"
+      response:
+          value: ["Hot", "Warm", "Freezing"]
+
+    - label: "Write selected temperature level"
+      command: "writeAttribute"
+      attribute: "SelectedTemperatureLevel"
+      arguments:
+          value: 2
+
+    - label: "Read back selected temperature level"
+      command: "readAttribute"
+      attribute: "SelectedTemperatureLevel"
+      response:
+          value: 2
+
+    - label: "Write selected temperature level"
+      command: "writeAttribute"
+      attribute: "SelectedTemperatureLevel"
+      arguments:
+          value: 1
+
+    - label: "Read back selected temperature level"
+      command: "readAttribute"
+      attribute: "SelectedTemperatureLevel"
+      response:
+          value: 1
+
+    - label: "Write selected temperature level"
+      command: "writeAttribute"
+      attribute: "SelectedTemperatureLevel"
+      arguments:
+          value: 3
```

Comment:
This responds with an error, right?  This should have a "response:" bit that says what error is expected.

Reply:
Done